Assignment2/ziader-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,48 @@
+﻿{
+	"@context": {
+		"pop":"http://www.ejemplo.es/apo/catalogo/"
+	},
```

Comment:
Please, define in the context the attributes that will use and the corresponding URI.